bedevere/news.py

```diff
@@ -28,20 +27,12 @@
                                   description='"skip news" label found')
 
 
-@router.register('pull_request', action='opened')
-@router.register('pull_request', action='synchronize')
-async def check_news(event, gh, *args, **kwargs):
-    pr_number = event.data['number']
-    pull_request = event.data['pull_request']
-    # For some unknown reason there isn't any files URL in a pull request
-    # payload.
-    files_url = f'/repos/python/cpython/pulls/{pr_number}/files'
-    # Could have collected all the filenames in an async set comprehension,
-    # but doing it this way potentially minimizes the number of API calls.
+async def check_news(gh, pull_request, filenames=None):
```

Comment:
Add a comment that the route is handled through the `files` module.

Reply:
Added here and in `prtype`.